src/Operations.jl

```diff
@@ -1352,12 +1352,12 @@ function sandbox(fn::Function, ctx::Context, target::PackageSpec, target_path::S
             temp_ctx.env.project.deps[target.name] = target.uuid
             write_env(temp_ctx.env, update_undo = false)
             try
-                Pkg.resolve()
+                Pkg.resolve(;io=devnull)
```

Comment:
Does this really do anything? Isn't there a need for a 

```
    Context!(ctx; kwargs...)
```

in `Pkg.resolve` or something?

Reply:
The keywords get handed to `Pkg.up`, which makes the call to `Context!`. Perhaps it would be better to make the call explicitly in `resolve` for clarity?

Reply:
Maybe, anyway, let's get this in.